docs/source/model_design/vivarium_model_components/observers/index.rst

```diff
@@ -0,0 +1,386 @@
+    - 76.7
+    - **Dead**
+  * - 3
+    - Female
+    - 52.0
+    - Alive
+  * - 4
+    - Male
+    - 98.4
+    - **Dead**
+  * - 5
+    - Female
+    - 71.3
+    - **Dead**
+  * - 6
+    - Female
+    - 38.2
+    - Alive
+
+Now let's advance one additional year to 1/1/2027. Here we see that one additional simulant has died and the surviving simulants have aged another year.
+
+.. list-table:: Population table, time 1/1/2027
+  :header-rows: 1
+
+  * - Simulant
+    - Sex
+    - Age (current or at time of death)
+    - Vital status
+  * - 1
+    - Male
+    - 27.2
+    - Alive
+  * - 2
+    - Male
+    - 76.7
+    - **Dead**
+  * - 3
+    - Female
+    - 52.0
+    - **Dead**
+  * - 4
+    - Male
+    - 98.4
+    - **Dead**
+  * - 5
+    - Female
+    - 71.3
+    - **Dead**
+  * - 6
+    - Female
+    - 39.2
+    - Alive
+
+While it is fairly easy for us to track information about deaths among our simulants in this example by comparing the population tables, this would quickly become inefficient as the number of simulants, timesteps, and measures to track increases. To avoid having to save out the entire population table at various timepoints to track what is happening in the simulation, observers can more efficiently track specific information for us.
+
+An observer to track deaths that occurred during the entire simulation (1/1/2025-1/1/2027) would record the total number of deaths that occurred across all timesteps: equal to 4 deaths in this example.
+
+Age and sex are standard stratifications for vivarium observers. However, if we would like to examine observed data specific to any of our additional simulated attributes, we can stratify our them by such factors.
+
+  * For instance, a death observer stratified by sex would return:
+
+    - Males: 2 deaths
+    - Females: 2 deaths
+
+  * A death observer stratified by sex and year would return:
+
+    * Males, 2025: 2 
+    * Males, 2026: 0
+    * Females, 2025: 1
+    * Females, 2026: 1
+
+General guidelines
+------------------
+
+Determining which measures to observe
++++++++++++++++++++++++++++++++++++++
+
+A good place to start in determining what observations you will need for a given simulation is to make a list of all of the measures you will need in order to evaluate the simulation's verification and validation criteria (consult the V&V criteria sections in the relevant component model documents included in your simulation) and the overall results of interest for the simulation (often measures like DALYs averted between scenarios, but this will depend on your specific research question).
+
+Once you have this list, remember that simulation observers record event **counts** rather than rates, so decompose all of the rates into the count measures of the numerators and denominators. For instance, in order to obtain the all-cause mortality rate (ACMR) in a simulation, you will need observations of deaths counts (the ACMR numerator) and person time counts (the ACMR denominator).
```

Comment:
```suggestion
Once you have this list, remember that simulation observers record event **counts** rather than rates, so decompose all of the rates into the count measures of the numerators and denominators. For instance, in order to obtain the all-cause mortality rate (ACMR) in a simulation, you will need observations of death counts (the ACMR numerator) and person time counts (the ACMR denominator).
```